dist/index.js

```diff
@@ -45,6 +52,15 @@ var HCaptcha = function (_React$Component) {
     _this.onerrorCaptcha = _this.onerrorCaptcha.bind(_this);
     _this.onsubmitCaptcha = _this.onsubmitCaptcha.bind(_this);
     _this.closeCaptcha = _this.closeCaptcha.bind(_this);
+    _this.resetCaptcha = _this.resetCaptcha.bind(_this);
+
+    // https://hcaptcha.com/docs/languages lists available codes.
+    _this.languageOverride = props.languageOverride;
+    // custom endpoint
+    _this.endpoint = props.endpoint;
+
+    // https://hcaptcha.com/docs/languages lists available codes.
+    _this.languageOverride = props.languageOverride;
```

Comment:
this is also in line 58

Reply:
it's also so in the `src/` version which I did *not* touch. 
I ran build to do `dist/` and it includes changes from older commits.

But yeah this is not optimal. I'll remove from src too